### DISCUSSION TITLE
Sveltekit Edge

### DISCUSSION THREAD

**Discussion by wiesson** [vercel/community#573] [Help]
I got this error when deploying Sveltekit with edge and split true

```
Run npm run preview to preview your production build locally.
--
15:41:12.976 |  
15:41:12.976 | > Using @sveltejs/adapter-vercel
15:41:13.007 | ✔ done
15:41:13.023 | Error! TODO: merge `.vercel/output` dir when Root Directory is used
15:41:13.034 | Error: Command "vercel build" exited with 1
```

Anyone has seen this error? My app is not in the root folder, it's in `/apps/website`

**Comment by leerob** [comment]
Hey! This is still not rolled out to everyone - you can opt-in with `ENABLE_VC_BUILD=1` as an environment variable. Let me know if that works.

**Reply by wiesson** [reply]
I was able to deploy it with `vc deploy --prod` from the folder of the project. But not with automatic git deployment. I guess, my monorepo/turbo setup causes that, but I'm not sure. `ENABLE_VC_BUILD=1` is set. My config looks like this:
```
import preprocess from 'svelte-preprocess';
import adapter from '@sveltejs/adapter-vercel';

/** @type {import('@sveltejs/kit').Config} */
const config = {
  kit: {
    adapter: adapter({
      // edge: true,
      // split: true
    })
  },
  preprocess: [preprocess({ postcss: true })]
};

export default config;
```
Once I have disabled split and edge, the deployment was successful, but it looks like it runs on edge (displaying location and ip, same as Rich did in his example). Can I somehow check, if it runs with normal functions or edge?

**Reply by thenbe** [reply]
@wiesson  I _think*_ you can use the [`x-vercel-id`](https://vercel.com/docs/concepts/edge-network/headers#x-vercel-id) response header to check if it's using serverless functions or edge.

Since you already deployed Rich's [example](https://github.com/Rich-Harris/sveltekit-on-the-edge) repo, open your devtools and check that header. If it has 2 [regions](https://vercel.com/docs/concepts/edge-network/regions) (the one closest to you + the one [selected](https://vercel.com/docs/concepts/functions/regions#configuring-the-region) in your vercel project settings), then it's not using edge. Else, if it's just using the closest region to your location then that should mean it is deployed on the edge.

Note: The default selected region for new Vercel projects is Washington, DC. So if you do live on the US east coast, make sure to manually set to somewhere far away or else you'll aways get a single location in the header.

_*Note: I'm not sure if this is a correct way to check if it's running on Edge, this is only my current understanding from reading the docs._

**Comment by thenbe** [comment]
I get the same error message. I'm also using a monorepo setup with pnpm. To be fair, the error message does indicate that this will be fixed in the future. I guess monorepos will have to wait a bit to use this.
